addon/adapters/github.js

```diff
@@ -19,7 +19,8 @@ export default RESTAdapter.extend({
   }),
 
   pathForType(type) {
-    return Ember.String.camelize(Ember.String.pluralize(type.replace('github', '')));
+    return Ember.String.camelize(Ember.String.pluralize(type.replace('github', '')))
+      .replace('organizations', 'orgs');
```

Comment:
Why don't I restore it the way it was until we can come up with something better? This is the effect of a design intention that can't play out completely.